lib/maintenance_tasks.rb

```diff
@@ -64,25 +64,7 @@ module MaintenanceTasks
   mattr_accessor :backtrace_cleaner
 
   # @private
```

Comment:
I can't remember why this ended up with a `@private` tag. I think it should be `@!attribute` to match the others?

Reply:
Below you can see the `@!attribute` docs, so I just moved the accessor to it. I assume the method override was marked as private because you didn't want repeated docs.